### PR TITLE
Commented tags to define symbols

### PR DESCRIPTION
**Pull request by gautamdayal**

### CODE REVIEW

File: apertium-por.por.dix [hunk 1]
```diff
@@ -10,83 +10,83 @@
...
+    <sdef n="det"  c="Determiner"/>
+    <sdef n="dem"  c="Demonstrative"/>
+    <sdef n="def"  c="Definite"/>
+    <sdef n="cnjcoo"  c="Co-ordinating conjunction"/>
+    <sdef n="cnjsub"  c="Sub-ordinating conjunction"/>
+    <sdef n="cnjadv"  c="Conjunctive adverb"/>
+    <sdef n="nt"  c="Neuter"/>
+    <sdef n="vbser"  c="Verb "to be""/>
+    <sdef n="vbhaver"  c="Verb "to have""/>
+    <sdef n="vblex"  c="Standard ("lexical") verb"/>
```
Comment (mr-martian) [inline apertium-por.por.dix:56]: can't have double quotes inside xml attributes - change to single quotes

File: apertium-por.por.dix [hunk 2]
```diff
@@ -10,83 +10,83 @@
...
-    <sdef n="lpar" /> 
-    <sdef n="rpar" /> 
-    <sdef n="lquest" /> 
-    <sdef n="pmp" /> 
-    <sdef n="infps" /> 
-    <sdef n="cuavb" /> 
-    <sdef n="percent" /> 
-    <sdef n="web" /> 
+    <sdef n="acr"  c="Acronym"/>
+    <sdef n="predet"  c="	Pre determiner"/>
```
Comment (mr-martian) [inline apertium-por.por.dix:14]: remove extra space

**Review by mr-martian — CHANGES_REQUESTED** [review]

File: apertium-por.por.dix [hunk 3]
```diff
@@ -51,9 +51,9 @@
     <sdef n="cnjsub"  c="Sub-ordinating conjunction"/>
     <sdef n="cnjadv"  c="Conjunctive adverb"/>
     <sdef n="nt"  c="Neuter"/>
-    <sdef n="vbser"  c="Verb "to be""/>
-    <sdef n="vbhaver"  c="Verb "to have""/>
-    <sdef n="vblex"  c="Standard ("lexical") verb"/>
+    <sdef n="vbser"  c="Verb 'to be'/>
+    <sdef n="vbhaver"  c="Verb 'to have'/>
```
Comment (mr-martian) [inline apertium-por.por.dix:55]: these two still need an ending double quote

**Review by mr-martian — CHANGES_REQUESTED** [review]

**Comment by gautamdayal** [comment]
Added double quotes at the end

**Review by mr-martian — APPROVED** [review]

**Pull request closed without merging**